rts/Sim/Weapons/Weapon.cpp

```diff
@@ -1027,8 +1029,15 @@ bool CWeapon::TestTarget(const float3 tgtPos, const SWeaponTarget& trg) const
 bool CWeapon::TestRange(const float3 tgtPos, const SWeaponTarget& trg) const
 {
 	RECOIL_DETAILED_TRACY_ZONE;
-	const float heightDiff = tgtPos.y - aimFromPos.y;
-	const float targetDist = aimFromPos.SqDistance2D(tgtPos);
+
+	const float heightDiff = (rangefrombase > 0) 
```

Comment:
What if I want the diff to be zero, or maybe even negative (some flying unit?). In particular zero is a good salient default for a height bonus. Perhaps make it controlled by a separate bool?